browser/extensions/brave_component_loader.h

```diff
@@ -54,6 +54,7 @@ class BraveComponentLoader : public ComponentLoader {
       WILL_REMOVE,
     };
     virtual void OnPdfExtensionAction(PdfExtensionAction action) = 0;
+    virtual void OnComponentRegistered(std::string extension_id) = 0;
```

Comment:
Should this be a const reference just for consistency?

Reply:
I think it is consistent to `OnComponentRegistered` and `OnComponentReady` defined in this same file, so I'll leave it as is.